libs/wingsdk/test/target-tf-aws/__snapshots__/bucket.test.ts.snap

```diff
@@ -444,6 +444,14 @@ exports[`bucket with onCreate method 2`] = `
                 "id": "Code",
                 "path": "root/Default/Code",
               },
+              "ParameterRegistrar": {
+                "constructInfo": {
+                  "fqn": "constructs.Construct",
+                  "version": "10.3.0",
+                },
+                "id": "ParameterRegistrar",
+                "path": "root/Default/ParameterRegistrar",
+              },
```

Comment:
Not sure why these additions only just happened and weren't committed with the original PR to add platform parameter registration.